File: submissions/3462-vowels-game-in-a-string/solution.java
class Solution {
    public boolean doesAliceWin(String s) {
        for(int i=0; i < s.length(); i++){
            int vow = s.charAt(i);
            if (vow == 'a' || vow == 'e' || vow == 'i' || vow == 'o' || vow == 'u') {
                return true;
            }
        }
        return false;
        
    }
}

File: submissions/3462-vowels-game-in-a-string/solution.js
/**
 * @param {string} s
 * @return {boolean}
 */
var doesAliceWin = function(s) {
    for(let i=0; i<s.length; i++){
        vow = s[i];
        if(vow === 'a' || vow === 'e' || vow === 'i' || vow === 'o' || vow ==='u') return true;
    }
    return false;
};
